### PR TITLE
Submission for code review.

### PR DESCRIPTION
**Pull request by vzdrizhni**
Please review the Newsweek clone page made using bootstrap.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,790 @@
...
+}
+
+html,
+body {
+  height: 100%;
+  overflow: auto;
+}
+
+body {
+  background-color: #f4f5f6 !important;
```
Comment (pelzolga123) [inline style.css]: Use of `!important` is considered a bad practice, please remove it from your project.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Roman!  
### Your project looks :+1: amazing!
Still there are changes, you need to make, before this project can be approved:
* [x] Large version:
Bottom part is not aligned like on original website:
image
Original:
image
* [ ] Tablet version:
* `Editor's pick` section doesn't look like original one:
image
Original:
image
* Next sections should be centered properly, image inside `newsletter` part takes near 25% of div, when text takes the rest of it  :
image
Original:
image
* Footer should look like original:
image
Original:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*
* [ ] Mobile version:
* Sections logo looks too small:
image
Original:
image
* Next sections should be centered properly :
image
Original:
image
* Footer should look like original:
image
Original:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by vzdrizhni** [comment]
Thanks for the review. Fixed issues you showed me.
- Fixed footer section for large, medium and small screens.
- Fixed Footer.
- Removed !important properties.
- Fixed icons size.
Thanks to @pelzolga123 
Please review it again.

**Review by mosesogwo — APPROVED** [review]
**Hi @vzdrizhni, you have done well implementing the requested changes:+1:.**
**Your project is approved and you can now merge to master:heavy_check_mark:**

Here are some recommendations to make your page better: 
 - On large and small screens, the images in the **Opinion** section have different sizes. You should make them of the same size.
- Remove the vertical scroll from the **Top Story** section
- Add some margin between the footer section and the section above it.

**Cheers:stars:**

### Status: Approved!

**Comment by vzdrizhni** [comment]
Awesome! Thank you!

**Pull request merged**